openshift/templates/django.json

```diff
@@ -186,7 +187,13 @@
                     "name": "DJANGO_SECRET_KEY",
                     "value": "${DJANGO_SECRET_KEY}"
                   }
-                ]
+                ],
+                "livenessProbe": {
+                  "httpGet": {
+                    "path": "/_health/live",
+                    "port": "web"
```

Comment:
working with @ironcladlou it seems that currently this doesn't resolve correctly for the liveness probe and returns a port of 0.  work around is to specify the value of 'web'